packages/UI/src/components/organisms/Dropdown/Dropdown.tsx

```diff
@@ -47,7 +49,7 @@ const Dropdown = ({
         prev[text] = checked;
         return prev;
       });
-    recieveValue?.(checkedOptions);
```

Comment:
Where is `checkedOptions` defined?

Reply:
@chungthuang this is a stateful variable defined on line 55 of this file.